rust/ast-generator/src/main.rs

```diff
@@ -485,8 +485,6 @@ fn main() -> anyhow::Result<()> {
         .expect("Failed to parse grammar");
     let mut grammar = codegen::grammar::lower(&grammar);
 
-    grammar.enums.retain(|x| x.name != "Adt");
```

Comment:
Perhaps skip `VariantDef` completely, not sure what its use would be.
```suggestion
    grammar.enums.retain(|x| x.name != "VariantDef");
```

Reply:
good point, doesn't seem really relevant for the moment

Reply:
Done